src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/BaseRESTController.java

```diff
@@ -4,13 +4,15 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.security.Principal;
+
 @RestController
 @RequestMapping("/base")
```

Comment:
Wywalmy to... albo oznaczmy jakoś, że to jest testowe
A jeśli chcesz to zachować, ale nie chcemy tego mieć jako bean produkcyjny, to daj odpowiednią adnotację, która sprawi, ze ten bean powstanie tylko z profilem "local" (@Profle bodajze)
Ewentualnie @ConditionalOnProperty